scripts/install_ws_dependencies.sh

```diff
@@ -41,10 +41,13 @@ while [ : ]; do
         local_NODE_INSTALL=$2
         shift 2
         ;;
-    -u | --unify)
-        echo -n "${packages[@]}" >> "$apt_file_path"
-        echo -n "${externals[@]}" >> "$ext_file_path"
+    unify)
+        # the space is necessary to not conflict with the other install scripts
```

Comment:
You need to put this comment in each of the files not just the install_ws_dependencies one.